src/Hazelcast.Net.Tests/Clustering/FailoverTests.cs

```diff
@@ -286,7 +287,7 @@ public async Task TestClientCanFailover(bool smartRouting, int memberCount)
                         o.Networking.Addresses.Clear();
                         o.Networking.Addresses.Add(Cluster0Address);
                         o.Networking.ReconnectMode = Hazelcast.Networking.ReconnectMode.ReconnectAsync;
-                        o.Networking.SmartRouting = smartRouting;
+                        o.Networking.RoutingMode.Mode = smartRouting ? RoutingModes.AllMembers : RoutingModes.SingleMember; 
```

Comment:
`smartRouting ? RoutingModes.AllMembers : RoutingModes.SingleMember;` 

I see this logic a few places -- maybe use some helper for it? I understand why you're doing it w.r.t. old logic but new reader without the diff won't know.

Reply:
I've refactored the smart routing argument to `RoutingModes` directly, no middle transformation anymore.